latexify/core.py

```diff
@@ -163,9 +167,44 @@ def visit_Compare(self, node):
 
     if isinstance(node.ops[0], ast.Eq):
       return lstr + '=' + rstr
+    # CHANGE:(20200801@1MLightyears) Add support for conditions for <,>,\ge,\le,\ne,is
+    elif isinstance(node.ops[0], ast.Gt):
+      return lstr + '>' + rstr
+    elif isinstance(node.ops[0], ast.Lt):
+      return lstr + '<' + rstr
+    elif isinstance(node.ops[0], ast.GtE):
+      return lstr + r'\ge ' + rstr
+    elif isinstance(node.ops[0], ast.LtE):
+      return lstr + r'\le ' + rstr
+    elif isinstance(node.ops[0], ast.NotEq):
+      return lstr + r'\ne ' + rstr
+    elif isinstance(node.ops[0], ast.Is):
+      return lstr + r'\space is\space ' + rstr
+    elif isinstance(node.ops[0], ast.IsNot):
+      return lstr + r'\space is\space not\space ' + rstr
+
     else:
       return r'\operatorname{unknown\_comparator}(' + lstr + ', ' + rstr + ')'
 
+  # CHANGE:(20200801@1MLightyears) Add support for BoolOp, which allow Latexify to
+  # deal with more complex boolean statements.
+  # The structure is weak, for I seldom use very complex boolean statements and
+  # currently have no idea how to simply implement a result like "x=0,1" instead of
+  # "x=0 Or x=1" as it is now XD
+  def visit_BoolOp(self, node):
+    '''
+    When node is an instance of ast.BoolOp, it has:
+    node.op(ast.And, ast.Or, ast.Not): type of the boolean operator.
+    node.values(list of ast.Compare, ast.BoolOp): a list of boolean sub-clause.
+    '''
+    logic_operator = r' \operatorname{unknown\_comparator} '
+    if isinstance(node.op, ast.Or):
+      logic_operator = r'\\&\mathrm{Or}\space '
```

Comment:
LaTeX has logical binary operators `\lor` and `\land`.

Reply:
OMG I even never heard them!
Thank you for teaching me these. I will change them at the next commit.